challenges/objects-arrays.js

```diff
@@ -59,11 +73,20 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+
+for (i = 0; i < graduates.length; i++) {
+  contactInfo.push(`${graduates[i].first_name} ${graduates [i].email}`);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for (let i = 0; i < graduates.length; i++){
+  if (graduates[i].university.toLowerCase().includes("uni")){
```

Comment:
Think about how this modifies the university name and if this is the behavior you want based on the instructions!